ParseTwitterUtils/PF_Twitter.m

```diff
@@ -92,6 +92,30 @@ - (BFTask *)authorizeInBackground {
         return source.task;
     }];
 }
+- (BFTask *)deauthorizeInBackground {
+    if (self.consumerKey.length == 0 || self.consumerSecret.length == 0) {
+        //TODO: (nlutsenko) This doesn't look right, maybe we should add additional error code?
+        return [BFTask taskWithError:[NSError errorWithDomain:PFParseErrorDomain code:1 userInfo:nil]];
+    }
+
+    return [[self _performDeauthAsync] pftw_continueAsyncWithBlock:^id(BFTask *task) {
+        BFTaskCompletionSource *source = [BFTaskCompletionSource taskCompletionSource];
+        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
👍 will remove -- and confession: this was mostly copied from `authorizeInBackground`, which also has a `dispatch_async` to the main queue (and why theres a `//TODO` with your name in it, oops, heh). worth removing that `dispatch_async` at the same time, because of the same logic?